libs/sdk-backend-spi/src/common/settings.ts

```diff
@@ -130,6 +130,11 @@ export interface ISettings {
      */
     enableKPIDashboardDrillToURL?: boolean;
 
+    /**
+     * Enable multiple date dimensions (ONE-5043) - development feature flag
+     */
+    enableMultipleDatesDEV?: boolean;
```

Comment:
I think adding this is not necessary, since it is a feature flag the external users do not care about. The `[key: string]: number | boolean | string | object | undefined;` will catch this sou you can use this during testing no problem. I am just afraid this will slip into some public release.

Reply:
I guess that usen object and string as a key like this flags["enableMultipleDatesDEV"] is not good pattern ...

Reply:
Done [commit](https://github.com/gooddata/gooddata-ui-sdk/pull/1444/commits/c1702f517fdc480b3b0835a2a903ca209c9a880d)